synapseclient/table.py

```diff
@@ -473,7 +473,8 @@ class SchemaBase(Entity, Versionable):
     _property_keys = Entity._property_keys + Versionable._property_keys + ['columnIds']
     _local_keys = Entity._local_keys + ['columns_to_store']
 
-    @abstractproperty  #forces subclasses to define _synapse_entity_type
+    @property
+    @abstractmethod  #forces subclasses to define _synapse_entity_type
     def _synapse_entity_type(self):
         pass
 
```

Comment:
At first observation this seems non-pythonic.  I am sure I will change my mind when I get to the use of this.